sunxspex/tests/test_photon_power_law.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+import astropy.units as u
+
+from sunxspex import photon_power_law as ppl
+
+''' i know this isn't really a test but it shows that things are working '''
+
+
+def basic_plot_test():
+    paramz = dict()
+    paramz['energy_edges'] = (edges := np.logspace(0, 2, num=40) << u.keV)
+    paramz['reference_energy'] = 50 << u.keV
+    paramz['reference_flux'] = 10 << ppl.PHOTON_RATE_UNIT
+    paramz['break_energy'] = 20 << u.keV
+    paramz['lower_index'] = 1
+    paramz['upper_index'] = 6
+
+    flux = ppl.broken_power_law_binned_flux(**paramz)
+
+    fig, ax = plt.subplots(figsize=(8, 6))
+    ax.stairs(flux.value, edges.value)
```

Comment:
This is likely my own ignorance here but why assign an `edges` variable and use `edges.value` rather than just use `paramz['energy_edges'].value`?

Reply:
No real reason. :)